cmd/celestia-appd/cmd/root.go

```diff
@@ -255,23 +254,22 @@ func NewAppServer(logger log.Logger, db dbm.DB, traceStore io.Writer, appOpts se
 }
 
 func createAppAndExport(
-	logger log.Logger, db dbm.DB, traceStore io.Writer, height int64, forZeroHeight bool, jailWhiteList []string,
```

Comment:
IMO if the parameters to a function are going to span multiple lines, it's easier to read if they're each on a newline. Very minor code preference thing and I really with `gofmt` would just pick a style and enforce it consistently